.github/workflows/stale.yml

```diff
@@ -0,0 +1,32 @@
+name: "Close stale PRs"
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: >
+            Automated Message:
+            
```

Comment:
Maybe it makes sense to remove this empty line and the `Automated Message` note like in the PR message?